hashintel/hash
# hash.dev: switch to next-seo, add default OG image

## 🌟 What is the purpose of this PR?

The motivation for this PR is to add a default OG image to all pages, as currently only the individual blog pages have them.

It introduces `next-seo` as the easiest way of managing metadata defaults + overwrites.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203301853897838/1203358502199028/f) _(internal)_

## ❓ How to test this?



1.  View the index, blog index, and individual blog pages – check the elements in `<head>`
1.  Once the preview deployment is available, paste a link to pages in Slack (or wherever), check the preview
1.  Paste a link in a draft tweet
